### DISCUSSION TITLE
How do you delete the line above a table if it's at the beginning of a note? (Android)

### DISCUSSION THREAD

**Discussion by Frederick888** [streetwriters/notesnook#3015] [Q&A]
I just realised that you can insert a new line by double-tapping the area above the table if the table is at the beginning of the note.

Now I wonder how I can get rid of this line considering there's no Delete button on the Android virtual keyboard (I'm using Gboard)? If I move the cursor to the beginning of the first cell and hit Backspace, it removes the line but it immediately comes back.

**Comment by thecodrr** [comment]
There's no way to delete that first line aside on mobile. On PC, there's the delete key.

**Reply by Frederick888** [reply]
Gotcha. Should I create an issue or there's already one?

**Reply by thecodrr** [reply]
What would be the ideal solution though?

**Reply by Frederick888** [reply]
Personally I think it makes sense to delete that line if I hit Backspace when the cursor is at the beginning of the top-left cell.

Another solution is to allow having a taller cursor to the left of the whole table. But it's not very consistent with the small horizontal cursor we currently have when inserting lines between two adjacent tables.
